src/App.tsx

```diff
@@ -1,24 +1,45 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { getTodos } from './api/todos';
+import { Content } from './components/Content';
+import { Errors } from './components/Errors';
 import { UserWarning } from './UserWarning';
 
+import { Error } from './types/Error';
+import { Todo } from './types/Todo';
+
 const USER_ID = 0;
```

Comment:
As I wrote it before. Don't hardcode the userId, please. Check how it works in the working example. Please, if you have any questions, ask them in the fe_chat